Documentation/CHANGES_IN_V5.md

```diff
@@ -14,9 +14,9 @@ This new container will be dynamically built on the users machine. Users will ne
    `./metroae install vsds -vvv` use `./metroae-container install vsds -vvv`
    Another example for deployments other than defaults
    `./metroae install vsds specialdeployment -vvv` use `./metroae-container install vsds specialdeployment -vvv`
-4. For all image paths, make sure they start with `/metroae`. Here `/metroae` refers to the present working directory for the user.
+4. Image paths may begin with `./` instead of `/metroae`. Here `/metroae` refers to the present working directory for the user.
```

Comment:
We should update this to be more precise. Users can now specify file locations relative to the MetroAE container using ./ instead of /metroae - this isn't specific to image files.